presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/IcebergConfig.java

```diff
@@ -52,4 +54,18 @@ public IcebergConfig setCompressionCodec(HiveCompressionCodec compressionCodec)
         this.compressionCodec = compressionCodec;
         return this;
     }
+
+    @NotNull
+    public boolean isUniqueTableLocation()
+    {
+        return uniqueTableLocation;
+    }
+
+    @Config("iceberg.unique-table-location")
+    @ConfigDescription("If true UUID will be added to the table location")
```

Comment:
We don't need to say "if true" for boolean properties, as that's redundant. Let's phrase this as

> Include UUID in the table location

Reply:
looks really better, thanks